Dockerfile

```diff
@@ -107,10 +110,18 @@ RUN for PYBIN in /opt/python/*/bin; do \
 # **************************
 FROM install-pynini-from-wheel AS run-tests
 
+RUN echo "How does one run absl-py tests? We may never know the answer... Oh well."
+
 # Copy Pynini's tests and testing assets (but not Pynini itself)
-COPY --from=wheel-building-env /src/pynini-${PYNINI_VERSION}/pynini_test.py /tests/pynini_test.py
-COPY --from=wheel-building-env /src/pynini-${PYNINI_VERSION}/testdata /tests/testdata
+COPY --from=wheel-building-env /src/pynini-${PYNINI_VERSION}/tests /tests
+
+# RUN curl https://copr.fedorainfracloud.org/coprs/vbatts/bazel/repo/epel-7/vbatts-bazel-epel-7.repo > /etc/yum.repos.d/vbatts-bazel-epel-7.repo \
+#     && yum install -y bazel3
 
 # Run Pynini's tests for each of our Pythons
-RUN cd /tests && for PYBIN in /opt/python/*/bin; do \
-    "${PYBIN}/python" pynini_test.py -v -f || exit 1; done
+RUN cd / && for PYBIN in /opt/python/*/bin; do \
+    "${PYBIN}/pip" install absl-py || exit 1; \
```

Comment:
It may be better to do this setup before the tests (i.e., in its own `RUN` command outside of this loop). Rationale: debugging will be easier if the `pip` fails.